src/Uno.UI/UI/Xaml/Controls/ContentPresenter/ContentPresenter.cs

```diff
@@ -1109,6 +1118,10 @@ protected override Size MeasureOverride(Size size)
 				)
 			);
 
+#if UNO_SUPPORTS_NATIVEHOST
+			measuredSize = MeasureNativeElement(measuredSize);
```

Comment:
Should we do this only when `IsNativeHost` and also avoid the `base.MeasureOverride` in that case ?

Reply:
The native base measure is still needed for some measure considerations (some native controls don't measure properly). I may revisit this once other controls are used.